internal/pkg/aws/ecr/ecr.go

```diff
@@ -170,7 +170,7 @@ func (s Service) DeleteImages(images []Image, repoName string) error {
 
 // ClearRepository orchestrates a ListImages call followed by a DeleteImages
 // call to delete all images from the input ECR repository name.
-func (s Service) ClearRepository(repoName string) error {
+func (s Ecr) ClearRepository(repoName string) error {
```

Comment:
nit: the receiver is `s` whereas for other methods it's `c`. Can we update this one too please?